src/main/kotlin/blackjack/domain/card/CardDeck.kt

```diff
@@ -0,0 +1,25 @@
+package blackjack.domain.card
+
+import blackjack.domain.ExceptionTypes.WRONG_CARD_SIZE
+import blackjack.domain.card.suit.CardSuit
+import blackjack.domain.card.suit.SuitTypes
+
+object CardDeck {
+    private const val FULL_SIZE_OF_DECK = 52
+    private val cardList: List<Card>
+
+    init {
+        val newCardList: MutableList<Card> = mutableListOf()
+        SuitTypes.values()
+            .map { createCards(it.types) }
+            .forEach(newCardList::addAll)
```

Comment:
```suggestion
        val newCardList = SuitTypes.values()
            .flatMap { createCards(it.types) }
```
이 부분은 flatMap을 활용해볼 수 있을 것 같아요 🙂